content/foundations/foundations_survey_methods.rst

```diff
@@ -319,3 +319,4 @@ Survey methods not discussed on this page include:
  - Radiometric methods, which involve investigating the radioactivity of ground materials
  - Natural source and controlled source magnetotelluric methods (which involve low frequency EM sources)
  - Spontaneous Potential methods, which involve measuring natural voltages that occur in the ground due to movement of fluids or chemical reactions between fluids and minerals
+ - Nuclear magnetic resonance imaging, which uses the interaction of time-varying magnetic fields with proton spins to measure the amount of non-bound water. 
```

Comment:
Thanks! Good call - it would be great to expand the GPG to include content on NMR